docs: add tutorial to docs (#605)
bandada-infra/bandada: apps/docs/docs/tutorials/api-sdk/credential-group.md

Create a multiple-credentials group using the API SDK

```ts
const apiKey = "your-api-key"

const credentials = {
    credentials: [
        {
            id: "BLOCKCHAIN_TRANSACTIONS",
            criteria: {
                minTransactions: 10,
                network: "Sepolia"
            }
        },
        {
            id: "BLOCKCHAIN_BALANCE",
            criteria: {
                minBalance: "5",
                network: "Sepolia"
            }
        }
    ],
    expression: ["", "and", ""]
}

const groupCreationDetails = {
    name: "Multiple Credentials Group",
    description: "This is a description",
    treeDepth: 16,
    fingerprintDuration: 3600,
    credentials
}

const credentialGroup = apiSdk.createGroup(groupCreationDetails, apiKey)
```
More details on the credentials can be found at the [Credentials library](https://github.com/bandada-infra/bandada/tree/main/libs/credentials).

## Add members to a group

### Manual group

You can add users to a manual group by using the API SDK.

```ts
const groupId = "your-group-id"
const memberId = "member-id-1"
const apiKey = "your-api-key"

await apiSdk.addMemberByApiKey(
    groupId,
    memberId,
    apiKey
)
```

> ### Credential and multiple credentials group

Reviewer: ```suggestion
### Credential group
```

> You can invite users to join a credential and multiple credentials group by generating the credential group join URL.

Reviewer: ```suggestion
You can invite users to join a credential group by generating the credential group join URL.
```

Reviewer: This function only works for credential groups. We need to update that function or create a new one to add support to add members to a multiple credential group since the URL is different for both. 

(This will be part of a new issue/PR)

The url for multiple credential groups is: 

```bash
<dashboard-url>/credentials?group=<group-id>&member=<member-id>&type=multiple
```

Example: 

```bash
http://localhost:3001/credentials?group=79609564395766533586300841856737&member=15316025723065452012129624296669307962292967176408402265701167251062457738490&type=multiple
```

More info in the description of this PR: https://github.com/bandada-infra/bandada/pull/530

> ---

Reviewer: I think this file can be called `groups` since it covers group operations with the API SDK.

```

#### Get invite

Returns a specific invite along with the group details associated with the invite.

```ts
const inviteCode = "INVITECODE"
const apiKey = "your-api-key"

const invite = await apiSdk.getInvite(inviteCode)
```

#### Add member using an invite code

Adds a member to a group using an invite code.

```ts
const groupId = "your-group-id"
const apiKey = "member-id-1"
const inviteCode = "INVITECODE"

await apiSdk.addMemberByInviteCode(groupId, memberId, inviteCode)
```

#### Full example

This is an example of how the whole code would look like:

```ts
const groupId = "your-group-id"
const memberId = "member-id-1"
const apiKey = "your-api-key"

// generate invite code
const inviteCode = await apiSdk.createInvite(groupId, apiKey)

// wrap the invite code into a custom URL for easier sharing
const inviteLink = `https://client.bandada.pse.dev?inviteCode=${inviteCode.code}`

// upon receiving the invite link, validate the invite code
const invite = await apiSdk.getInvite(inviteCode.code)

// add member to group by invite code
const response = await apiSdk.addMemberByInviteCode(
    invite.group.id, 
    memberId, 
    inviteCode.code
)
```

> You can also refer to the [Client app](https://github.com/bandada-infra/bandada/blob/main/apps/client/src/pages/home.tsx) for more example of the add member to group by invite code logic.

Reviewer: ```suggestion
You can refer to the [Client app](https://github.com/bandada-infra/bandada/blob/main/apps/client/src/pages/home.tsx) to better understand the context and logic for adding members to a group using an invite code.
```
